docs/src/managed/managedcluster.md

```diff
@@ -0,0 +1,93 @@
+# Managed Clusters (OKE)
+- **Feature status:** Experimental
+- **Feature gate:** OKE=true,MachinePool=true
+
+Cluster API Provider for OCI (CAPOCI) experimentally supports managing OCI Container 
+Engine for Kubernetes (OKE) clusters. CAPOCI implements this with three
+custom resources:
+- OCIManagedControlPlane
+- OCIManagedCluster
+- OCIManagedMachinePool
```

Comment:
NIT: wrap each of the resources in a `` `example OCIManagedControlPlane`

Reply:
fixed